integration/listener_test.go

```diff
@@ -1,27 +1,23 @@
-package pubsub
+package integration
 
 import (
 	"context"
-	"errors"
-	"os"
 	"strings"
 	"time"
 
 	servicebus "github.com/Azure/azure-service-bus-go"
+	"github.com/keikumata/azure-pub-sub/listener"
 	"github.com/keikumata/azure-pub-sub/message"
 )
 
 // TestCreateNewListenerFromConnectionString tests the creation of a listener with a connection string
-func (suite *serviceBusSuite) TestCreateNewListenerWithConnectionString() {
-	listener, err := createNewListenerWithConnectionString()
-	if suite.NoError(err) {
-		connStr := os.Getenv("SERVICEBUS_CONNECTION_STRING")
-		suite.Contains(connStr, listener.namespace.Name)
-	}
+func (suite *serviceBusSuite) TestCreateNewListener() {
+	_, err := listener.New(suite.listenerAuthOption)
+	suite.NoError(err)
 }
 
 func (suite *serviceBusSuite) TestListenWithDefault() {
-	listener, err := createNewListenerWithConnectionString()
+	listener, err := listener.New(suite.listenerAuthOption)
```

Comment:
no duplicate tests, same suite for each auth type.

Reply:
awesome!